app/recommenders/hybrid_cf.py

```diff
@@ -47,6 +47,10 @@ class HybridCFRecommender:
     """
     Hybrid Collaborative Filtering recommender
     """
+    # 230 Mb in the storage utilizes ~30% of RecAPI memory (8GB) (!)
+    # after loading objects take ~650MB per worker x2 = 1300MB + some leakage
+    MAX_STORAGE_ARTIFACTS_SIZE = 500 * 1024 * 1024  # 500Mb
```

Comment:
as discussed with @eu9ene , my recommendation is to bump this up to 600 / 700 MB on the safer side. Thanks!